openpype/hosts/max/api/lib.py

```diff
@@ -42,6 +42,7 @@ def imprint(node_name: str, data: dict) -> bool:
             rt.SetUserProp(node, k, f"{JSON_PREFIX}{json.dumps(v)}")
         else:
             rt.SetUserProp(node, k, v)
+            print(k)
```

Comment:
Probably forgotten debug print?
```suggestion
```

Reply:
It was resolved, so it's not forgotten?

Reply:
Sorry I forgot....It's already been removed in the updated commit